js/plates-view/render-cross-section.ts

```diff
@@ -424,7 +418,8 @@ class CrossSectionRenderer {
       const p2tmp = p2.clone().lerp(p3, currentThickness2);
       const p3tmp = p2.clone().lerp(p3, currentThickness2 + rl.relativeThickness2);
       const p4tmp = p1.clone().lerp(p4, currentThickness1 + rl.relativeThickness1);
-      if (this.fillPath(getRockCanvasPattern(this.ctx, rl.rock), p1tmp, p2tmp, p3tmp, p4tmp)) {
+      const color = (field.normalizedAge || 0) < 0.8 ? "#333" : getRockCanvasPattern(this.ctx, rl.rock);
```

Comment:
Given that this same expression occurs here and in line 399 above, you could DRY up the code a bit with a function like `getRockCanvasPatternGivenNormalizedAge()` but it's not a big deal either way.

Reply:
Done in the last commit, thanks.